### PR TITLE
Add a Dice game #138

### LINKED ISSUE #138: Add a Dice game
**Describe the project you want to add with tech stack**

It will update the displayed number to simulate rolling a dice. You can further enhance this example by adding animations and other features to make it more interactive.
**Expected behavior**

**Screenshots (optional)**

dice

**Additional context (optional )**

### PR DESCRIPTION
**Pull request by shivam200446**
# Title and Issue number 

Title : Add a Dice game

Issue No. : #138

Code Stack: HTML, CSS , HTML

Close #138

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have updated the Index.html file for my contribution

### CODE REVIEW

File: native/machine-coding/Dice Game/dice.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+var randomnum1 = Math.floor(Math.random() * 6) + 1;
+var randomimg = "dice" + randomnum1 + ".png";
+var imgsource1 = "images/" + randomimg;
+var image1 = document.querySelectorAll("img")[0].setAttribute("src", imgsource1);
+
+var randomnum2 = Math.floor(Math.random() * 6) + 1;
+var imgsource2 = "images/" +"dice" + randomnum2 + ".png";
+var image2 = document.querySelectorAll("img")[1].setAttribute("src", imgsource2);
```
Comment (sadanandpai) [inline native/machine-coding/Dice Game/dice.js:8]: use const or let

File: native/machine-coding/Dice Game/index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Dicee</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href="https://fonts.googleapis.com/css?family=Indie+Flower|Lobster" rel="stylesheet">
```
Comment (sadanandpai) [inline native/machine-coding/Dice Game/index.html:7]: import the header as done in [counter](https://github.com/sadanandpai/frontend-mini-challenges/blob/main/native/machine-coding/counter/index.html)

Also, remove the other tags

File: native/machine-coding/Dice Game/styles.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
+/* Reset some default styles */
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  background-color: #1f1f1f;
+  color: #fff;
+}
```
Comment (sadanandpai) [inline native/machine-coding/Dice Game/styles.css:8]: dont modify font-family, margin, padding

**Review by sadanandpai — CHANGES_REQUESTED** [review]

**Review by shivam200446 — COMMENTED** [review]
@sadanandpai I have made all the changes you asked for. Thank you soo much for the Guidance and instruction.

**Comment by sadanandpai** [comment]
The challenge has to be added to the challenges list for it to appear in the UI @shivam200446

**Comment by shivam200446** [comment]
> The challenge has to be added to the challenges list for it to appear in the UI @shivam200446

How do it?

**Comment by sadanandpai** [comment]
Please refer to the contributing.md and other challenges

**Comment by shivam200446** [comment]
@sadanandpai Updated challenges.js please check it once.

**Comment by shivam200446** [comment]
Bhaiya, why it is showing conflict

File: native/helpers/challenges.js [hunk 4]
```diff
@@ -423,6 +423,14 @@ export const challenges = [
     developer: 'hritik',
     tags: [],
   },
+  {
+    title: 'Dice game',
+    link: 'Dice Game',
+    difficulty: 'easy',
+    developer: 'shivam200446',
```
Comment (sadanandpai) [inline native/helpers/challenges.js:430]: add your details to the contributors list also. Else name won't appear

**Review by sadanandpai — CHANGES_REQUESTED** [review]

**Comment by sadanandpai** [comment]
conflicts are there. Rebase with latest changes

**Comment by shivam200446** [comment]
@sadanandpai Bhaiya, I have updated all the changes and solved the conflicts. please look at it. Thank you so much for all your help really learned a lot.

**Review by sadanandpai — APPROVED** [review]

**Pull request merged**